libraries/networking/src/PacketReceiver.cpp

```diff
@@ -463,7 +463,4 @@ void PacketReceiver::handleVerifiedPacket(std::unique_ptr<udt::Packet> packet) {
         // insert a dummy listener so we don't print this again
         _packetListenerMap.insert(nlPacket->getType(), { nullptr, QMetaMethod() });
```

Comment:
We do, it's locked before the if.
Only thing I did here is remove unlocks before the end of the scope because they aren't needed.